CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# NEXT RELEASE
+# 12.9.0 Release notes
 
 ### Enhancements
 * <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))
```

Comment:
Now is a good time to remove this boilerplate and the one at the top of the _Fixed_ section as well.

Reply:
Thank you - I just removed it